### PR TITLE
chore(); eslint; update / fix _reference_

### PR DESCRIPTION
**Pull request by dmelotik**
Reducet the scope of `no-magic-numbers` to prevent linting overload.

Fix the issues in _reference_ so people can copy/paste the latest code without issues

### CODE REVIEW

File: subgraphs/_eslint-rules/no-string-literals.js [hunk 1]
```diff
@@ -19,9 +19,20 @@ module.exports = {
...
-      const ALLOWED_STRING_METHODS = ["concat"];
+      const ALLOWED_METHODS = [
+        "fromString",
+        "fromHexString",
+        "replace",
+        "concat",
+        "fromUTF8",
+        "readValue",
+        "undefined",
+        // these are for logs (note, debug is not allowed in master)
```
Comment (jaimehgb) [inline subgraphs/_eslint-rules/no-string-literals.js:30]: 🔥

**Review by jaimehgb — APPROVED** [review]
Other than the naming thing, all good 💪

File: subgraphs/_reference_/src/sdk/protocols/bridge/chainIds.ts [hunk 2]
```diff
@@ -11,20 +13,20 @@ export function chainIDToNetwork(chainID: BigInt): Network {
...
-  chainIDs.set(chainID, network);
-  reverseChainIDs.set(network, BigInt.fromU64(chainID));
+  CHAIN_IDS.set(chainID, network);
+  REVERSE_CHAIN_IDS.set(network, BigInt.fromU64(chainID));
 }
 
-export const chainIDs = new TypedMap<u64, Network>();
-export const reverseChainIDs = new TypedMap<Network, BigInt>();
+export const CHAIN_IDS = new TypedMap<u64, Network>();
+export const REVERSE_CHAIN_IDS = new TypedMap<Network, BigInt>();
```
[the remark this answers is not in the document]
Comment (dmelotik) [reply subgraphs/_reference_/src/sdk/protocols/bridge/chainIds.ts:29]: Maybe we should just make the var names one a warning, so it doens't yell at contributors too much..

I would like to keep the folder/filename one though, this is not hard to fix/follow.

Would we want to revert the changes to var names I made here then?

[the remark this answers is not in the document]
Comment (jaimehgb) [reply subgraphs/_reference_/src/sdk/protocols/bridge/chainIds.ts:29]: We can try it as a warning, but I'm not confident people will resolve them 😅 

Ok to folder/filename.

> Would we want to revert the changes to var names I made here then?

I'd personally revert them. But up to you. Although breaking, is not a very big deal in our case.

**Review by jaimehgb — APPROVED** [review]

**Review by tnkrxyz — COMMENTED** [review]
Would it be better to separate this into two PRs?

File: subgraphs/_reference_/src/sdk/protocols/bridge/chainIds.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (dmelotik) [reply subgraphs/_reference_/src/sdk/protocols/bridge/chainIds.ts:29]: Ah, okay. So to be clear, just get rid of the naming one? I might just have to make a new PR

**Comment by dmelotik** [comment]
Moved to #2088

**Pull request closed without merging**